.github/workflows/oracle.yml

```
@@ -111,19 +110,12 @@ jobs:
         run: |
           git clone $GITHUB_SERVER_URL/$GITHUB_REPOSITORY.git -b ${BRANCH_NAME} .
 
-      - name: Checkout Xpress linux
-        uses: actions/checkout@v2
-        with:
-          repository: rte-france/xpress-mp
-          path: ${{ env.XPRESS_INSTALL_DIR }}
-          ref: "9.0"
-          token: ${{ secrets.ACCESS_TOKEN }}
-
-      - name: set Xpress variables
+      - name: Set-up Xpress with pip
         run: |
-          cd ${{ env.XPRESS_INSTALL_DIR }}
-          echo "XPRESSDIR=$PWD" >> $GITHUB_ENV
-          echo "XPAUTH_PATH=$PWD/bin/xpauth.xpr" >> $GITHUB_ENV
+          python3 -m pip install xpress==8.13.8
```

I think we can try version 9.0, Oracle has python 3.9 by default.

```suggestion
          python3 -m pip install xpress==9.0.0
```

---

no it is not available for Python 6 on Oracle Linux
I set version 8.13.8 because it is the latest available version